netket/sampler/rules/gaussian_continuous.py

```diff
@@ -0,0 +1,31 @@
+import jax
+import jax.numpy as jnp
+
+from flax import struct
+
+
+from ..metropolis import MetropolisRule
+
+@struct.dataclass
+class GaussianRule(MetropolisRule):
+    r"""
+    A transition rule acting on all particle positions at once.
+
+    New proposals of particle positions are generated according to a Gaussian distribution of width s.
+    """
+    s: float = 1.
```

Comment:
by the way... `s` ain't a great choice for me.
Can't we have something like `sigma` or `variance`?